frontend/src/components/App.js

```diff
@@ -1,16 +1,50 @@
 import React, { Component } from 'react';
 import './App.css';
+import PaypalButton from './components/PayPalButton';
+import dotenv from 'dotenv'
+require('dotenv').config();
+dotenv.config();
+
+const CLIENT = {
```

Comment:
We may have other clients, so please change this to PAYPAL_CLIENT. Also, this should probably be instantiated in the component.